spatialdata/_core/_spatialdata.py

```diff
@@ -1139,12 +1147,14 @@ def bounding_box(
             min_coordinate=min_coordinate,
             max_coordinate=max_coordinate,
             target_coordinate_system=target_coordinate_system,
+            filter_table=filter_table,
         )
 
     def __call__(self, request: BaseSpatialRequest) -> SpatialData:
         from spatialdata._core._spatial_query import BoundingBoxRequest
 
         if isinstance(request, BoundingBoxRequest):
-            return self.bounding_box(**request.to_dict())
+            # TODO: filter table is on by default, shall we put filter_table inside the request?
+            return self.bounding_box(**request.to_dict(), filter_table=True)
```

Comment:
I think it makes sense to add `filter_table` to the request

Reply:
Ah forgot to specify: `filter_table` would be used only when the queried object is a `sdata`, and it would be ignored when we query an element. This is why I didn't add it in the request class. Shall I still add it?

Reply:
Ah yeah, good point. Let's not add `filter_table` to the request. Perhaps we can make `filter_table` a kwarg on `__call__`? Alternatively, we can just punt and deal with it later. I'm fine either way!

Reply:
Ok thanks, putting a `kwargs` in `__call__` leaving a `TODO`